### ISSUE TITLE
Ubuntu PPA needs Zesty release

### ISSUE THREAD

**Issue by Ads20000** [flatpak/flatpak#773]
I'm getting the following error trying to use the Flatpak Zesty (Ubuntu 17.04) source:

```
E:The repository 'http://ppa.launchpad.net/alexlarsson/flatpak/ubuntu zesty Release' does not have a Release file.
```
@alexlarsson please can you do a Zesty release of Flatpak to [the PPA](https://launchpad.net/~alexlarsson/+archive/ubuntu/flatpak)? Thanks!

As a bonus, so people don't run into this when 17.10 is released, maybe at some point before October do an Artful (17.10) release? If you can, do it now, so people running the development branch can still have updated Flatpak direct from upstream.

Alternatively, I'm not a massive fan of PPAs, maybe we could get Flatpak running inside Snappy...no wait that's probably not possible xD

**Comment by Conan-Kudo** [comment]
Flatpak is already in Ubuntu zesty. It's been included in Ubuntu since yakkety.

**Comment by Ads20000** [comment]
@Conan-Kudo Oh, is it kept updated (SRU'd) there though? Even if it's the current version at the moment, if it's not kept updated (so people would need the PPA on next Flatpak release), then really we need a release of the package in the repos in the PPA too since otherwise everyone runs into that error after upgrading to 17.04 or installing 17.04 and can't update until they disable the repo (admittedly all PPAs are disabled on upgrade to 17.04 so they would have to enable the 17.04 PPA repo, then find they can't update, then revert it to a 16.10 PPA repo or disable it entirely).

tl;dr Unless Flatpak is SRU'd, a Zesty PPA release of Flatpak would be good.

**Comment by alexlarsson** [comment]
Yeah, the plan is to add it to the PPA. Just a lack of time.

**Comment by Ads20000** [comment]
Fair enough, thought I'd file an issue because it is an issue! :P

**Comment by Ads20000** [comment]
I think there's been packages there for a while now, thanks Alex!

**Comment by hackel** [comment]
And now, just as you predicted, the Artful release is imminent and we have no artful builds.  Luckily the zesty build still seems to be working fine.  Just a heads-up @alexlarsson.  I'm sure you're wishing you could distribute flatpak with flatpak right about now!  Hey, maybe you can release flatpak as a snap? ;)

**Comment by Ads20000** [comment]
@probonopd has said it would be nice to see `appimaged` and AppImageUpdate as snaps. I'm sure it would be cool to see `snapd` as a Flatpak too. Problem is that this stuff takes work xD

**Comment by Conan-Kudo** [comment]
@Ads20000 AppImage has no confinement mechanism, whereas Flatpak and Snap do, so this makes things really hard.

**Comment by probonopd** [comment]
@Ads20000 it should be no problem to make snaps for appimaged and AppImageUpdate, but discussing any of this in the Flatpak issue tracker feels... a bit awkward ;-)
